src/main/java/com/entropyteam/entropay/employees/models/Status.java

```diff
@@ -0,0 +1,5 @@
+package com.entropyteam.entropay.employees.models;
+
+public enum Status {
+    APPROVED, REQUESTED
```

Comment:
what if a PTO is declined? for now all are approved but i think maybe an employee wants to cancel the pto

Reply:
yes, you're right